ios/BurntModule.swift

```diff
@@ -180,7 +203,15 @@ public class BurntModule: Module {
     Name("Burnt")
 
     AsyncFunction("toastAsync") { (options: ToastOptions) -> Void in
-      let view = SPIndicatorView(title: options.title, message: options.message, preset: options.preset.toSPIndicatorPreset())
+      let view : SPIndicatorView
+      if(options.preset == .none){
```

Comment:
formatting: `if (` is missing a space. Same with `.none) {`

Reply:
I notice this throughout. I wonder if there is a swift-format solution we can use.

Reply:
yeah, I use this lib https://github.com/nicklockwood/SwiftFormat, but I forget to run it.